src/Utilities/LoggingHelper.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+namespace Momento\Utilities;
+
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\NullHandler;
+use Monolog\Handler\StreamHandler;
+use Monolog\Level;
+use Monolog\Logger;
+
+class LoggingHelper
```

Comment:
not a blocker for this PR, but just want to make sure that we are thinking about we can ensure that the user's pre-existing logger/config will be respected if they have one.  If that's possible.  I'm not sure what the PHP logging ecosystem looks like.

Reply:
Good to call out, thanks. Monolog is one of a large number of logging packages for PHP. I chose it partly because it is also integrated with or (supported via plugin) by all of the main PHP app frameworks. 

There is a logging PSR, PSR-3, that Monolog supports and that may allow me to generalize what we can accept from users. I'll do some looking.